scripts/change-event-members-from-backers-to-attendees.js

```diff
@@ -0,0 +1,27 @@
+import MemberRoles from '../server/constants/roles';
+import models, { sequelize } from '../server/models';
+
+async function run() {
+  const members = await sequelize.query(
+    `SELECT m.* FROM "Members" m
+      INNER JOIN "Tiers" t ON m."TierId" = t.id
+      INNER JOIN "Collectives" c ON m."CollectiveId" = c.id
+    WHERE t."type" = 'TICKET'
+      AND m."role" != 'ATTENDEE'
+      AND c."type" = 'EVENT'
+    ORDER BY "createdAt" DESC`,
+    {
+      type: sequelize.QueryTypes.SELECT,
+      model: models.Member,
+      mapToModel: true,
+    },
+  );
+
+  for (const member of members) {
+    await member.update({ role: MemberRoles.ATTENDEE });
+  }
```

Comment:
This is quite heavy: we're going to load 3688 members then trigger 3688 update queries, while everything could be done in one single SQL statement. Not a requested change as the API can probably handle it, but in the future maybe try to do something like:

```sql
UPDATE "Members"
SET role = ...
FROM "Tiers", "Collectives"
WHERE ...
```

Reply:
@Betree : Yes, makes sense to create a migration script than a regular script. 👍🏽 Thanks for pointing it out. I've corrected that along with the SQL to use an update query.